opnode/contracts/Makefile

```diff
@@ -40,11 +49,29 @@ bindings-l1-block-info:
 		--type L1Block \
 		--out ./l1block/l1_block_info_raw.go
 
+bindings-L2ToL1MessagePasser:
```

Comment:
There are some mismatched names in files here, `withdrawer` -> `L2ToL1MessagePasser`

Reply:
TBH I'm ok with the go names being out of date, I've got an issue to update them (ENG-2195)